example/extensions/lib_api/init_lib.cc

```diff
@@ -28,7 +28,7 @@
 using namespace mxnet::ext;
 
 MXReturnValue initialize(int version) {
-  if (version >= 10900) {
```

Comment:
Do we have to update the required mxnet versions for examples?

Reply:
I was following #19196, where we updated the versions in the extensions examples. I could be wrong, but I guess we wanted to make sure the extensions only works with the current version or higher (no backward compatibility guaranteed)?